server/startup/serverRunning.coffee

```diff
@@ -1,17 +1,45 @@
+import fs from 'fs'
+import path from 'path'
+import semver from 'semver'
+
 Meteor.startup ->
 	oplogState = 'Disabled'
 	if MongoInternals.defaultRemoteCollectionDriver().mongo._oplogHandle?.onOplogEntry?
 		oplogState = 'Enabled'
 		if RocketChat.settings.get('Force_Disable_OpLog_For_Cache') is true
 			oplogState += ' (Disabled for Cache Sync)'
 
+	desiredNodeVersion = semver.clean(fs.readFileSync(path.join(process.cwd(), '../../.node_version.txt')).toString())
+	desiredNodeVersionMajor = String(semver.parse(desiredNodeVersion).major)
+
 	Meteor.setTimeout ->
 		msg = [
-			"     Version: #{RocketChat.Info.version}"
-			"Process Port: #{process.env.PORT}"
-			"    Site URL: #{RocketChat.settings.get('Site_Url')}"
-			"       OpLog: #{oplogState}"
-		].join('\n')
+			"Rocket.Chat Version: #{RocketChat.Info.version}"
+			"     NodeJS Version: #{'7.6.2'} - #{process.arch}"
+			"           Platform: #{process.platform}"
+			"       Process Port: #{process.env.PORT}"
+			"           Site URL: #{RocketChat.settings.get('Site_Url')}"
+			"   ReplicaSet OpLog: #{oplogState}"
+		]
+
+		if RocketChat.Info.commit?.hash?
+			msg.push("        Commit Hash: #{RocketChat.Info.commit.hash.substr(0, 10)}")
+
+		if RocketChat.Info.commit?.branch?
+			msg.push("      Commit Branch: #{RocketChat.Info.commit.branch}")
+
+		msg = msg.join('\n')
+
+		if semver.satisfies('7.6.2', desiredNodeVersionMajor)
```

Comment:
instead of `7.6.2` it should be comparing with the installed node version, right?